doc/book/link_routing.adoc

```diff
@@ -164,3 +164,37 @@ link-routing.
 The above configuration allows Rb and Rp to reject attaches that should
 be routed to B2 with an error message that indicates that there is no
 route available to the destination.
+
+Automatic Link Routes
+~~~~~~~~~~~~~~~~~~~~~
+
+There is an alternative way to use link routes that doesn't involve any
```

Comment:
If we decide that these link routes will not show up in the qdstat --linkRoute, should we add a comment here indicating that these link routes will not show up in the qdstat output?